staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2170,6 +2170,10 @@ type KubeVirtConfiguration struct {
 	WebhookConfiguration           *ReloadableComponentConfiguration `json:"webhookConfiguration,omitempty"`
 	ControllerConfiguration        *ReloadableComponentConfiguration `json:"controllerConfiguration,omitempty"`
 	HandlerConfiguration           *ReloadableComponentConfiguration `json:"handlerConfiguration,omitempty"`
+
+	// BackendStorageClass is the name of the storage class to use for the PVCs created to preserve VM state, like TPM.
+	// The storage class must support RWX in filesystem mode.
+	BackendStorageClass string `json:"backendStorageClass,omitempty"`
```

Comment:
To me `BackendStorageClass` sounds too generic and needs some kind of a prefix to explain what it's meant for.

Reply:
I agree. I've had a hard time naming this thing, suggestions are welcome!

Reply:
Maybe just `VMStateStorageClass`?

Reply:
Done